src/lib/mongoBackend/mongoQueryContext.cpp

```diff
@@ -376,8 +443,10 @@ HttpStatusCode mongoQueryContext
   /* First CPr lookup (in the case some CER is not found): looking in E-A registrations */
   if (someContextElementNotFound(rawCerV))
   {
+    LM_T(LmtForward, ("Calling registrationsQuery II"));
     if (registrationsQuery(requestP->entityIdVector, requestP->attributeList, &crrV, &err, tenant, servicePathV, 0, 0, false))
     {
+      // crrVectorPresent("registrationsQuery II", crrV);
```

Comment:
Same here.

Reply:
...and all the others (I have seen more below this line but I will not add new comments to avoid "review noise")

Reply:
Fixed in 3dc4bcf83f00f7937ce79cacd25417575f5beaed